client/src/App.tsx

```diff
@@ -1,5 +1,13 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import "./App.css";
+
+interface IUser {
```

Comment:
Pls use interfaces separately, in the interfaces folder. (Except DTO, that are stored in their own place)